ci/generate-npm-package.js

```diff
@@ -27,7 +27,7 @@ async function copyAssets() {
     await fs.ensureDir(folder);
 
     try {
-      const files = (await fs.readdir(builtPath)).filter((path) => !path.includes('patternlib'));
+      const files = (await fs.readdir(builtPath)).filter((path) => !path.includes('ds'));
```

Comment:
The rest of the changes are fine, Im just wondering if "ds" here might be a bit too generic and could cause issues if we have a folder at some point called something that has the letters "ds" in it. For example if there was a folder called "cards" this would pick it up. So I was thinking it would be good to at least change this to "ds-" which will help with this. Will also need to be changed on line 40

Reply:
updated